### DISCUSSION TITLE
how many channel I could create?

### DISCUSSION THREAD

**Discussion by hodubidu** [ant-media/Ant-Media-Server#5496] [FAQs]
Hello

I need to test weater Ant media WebRTC is appropriate for intergration into our Educational vedeo conferecing system.

Main issue :  I anticipate classrooms (channels) where 1~300people could join
   1. How many channels could we create and run simultaneously?
   2. How many people are supported in each channel?

**Comment by yashtandon113** [comment]
@hodubidu 
Thank you for your interest in Ant Media Server.

We understand your need to test Ant Media Server to ensure it's suitable for integration into your Educational Video Conferencing System. We appreciate your thorough evaluation process and are here to assist you every step of the way.

Let me answer your initial questions:

1. There is no limit on the server side for number of channels, users etc. It depends on the hardware specifications of the server.

2. There is no limit in this case as well. N number of users can join a single channel/conference room.

If one server itself cannot handle all the load, then cluster can also be deployed using multiple Ant Media Servers and different users can connect to the same conference room even if they are connected through different server in a backend. You can learn more about ways of scaling and clustering [here](https://antmedia.io/docs/category/clustering-and-scaling/).

Feel free to share your requirements or any details about the testing process you have in mind, and we'll do our best to provide the assistance you need.

**Reply by yashtandon113** [reply]
@alexaccandr 
Please check out the GitHub discussion [here](https://github.com/orgs/ant-media/discussions/5675) for more information regarding WebRTC viewer limit for a conference room to limit participants.

Although the new user's stream can still publish so for proper solution, we have created the GitHub issue as mentioned in the discussion.

I hope it helps. Please let us know if you have any questions.
